apollo-client/src/test/java/com/ctrip/framework/apollo/internals/RemoteConfigRepositoryTest.java

```diff
@@ -295,7 +295,7 @@ public Void answer(InvocationOnMock invocation) throws Throwable {
     when(pollResponse.getBody()).thenReturn(Lists.newArrayList(someNotification));
     when(someResponse.getBody()).thenReturn(newApolloConfig);
 
-    longPollFinished.get(5000, TimeUnit.MILLISECONDS);
+    longPollFinished.get(10000, TimeUnit.MILLISECONDS);
```

Comment:
why is it changed to 10 seconds?

Reply:
the the unit test failed by accident because of the cost of the initialization increased